dev/com.ibm.ws.concurrent_fat_cdi/fat/src/com/ibm/ws/concurrent/cdi/fat/ConcurrentCDITest.java

```diff
@@ -70,18 +76,26 @@ public static void setUp() throws Exception {
         ShrinkHelper.addDirectory(concurrentCDIWeb,
                                   "test-applications/concurrentCDIWeb/resources");
 
+        JavaArchive concurrentCDIEJBLib = ShrinkHelper.buildJavaArchive("concurrentCDIEJB", "concurrent.cdi.ejb.anno");
+
         JavaArchive concurrentCDIEJB = ShrinkHelper.buildJavaArchive("concurrentCDIEJB", "concurrent.cdi.ejb");
         ShrinkHelper.addDirectory(concurrentCDIEJB, "test-applications/concurrentCDIEJB/resources");
 
         EnterpriseArchive concurrentCDIApp = ShrinkWrap.create(EnterpriseArchive.class, "concurrentCDIApp.ear");
         concurrentCDIApp.addAsModule(concurrentCDIWeb);
         concurrentCDIApp.addAsModule(concurrentCDIEJB);
+        concurrentCDIApp.addAsLibraries(concurrentCDIEJBLib);
         ShrinkHelper.addDirectory(concurrentCDIApp, "test-applications/concurrentCDIApp/resources");
         ShrinkHelper.exportAppToServer(server, concurrentCDIApp);
 
+        // Create concurrentCDI4App.war
         // TODO Adding "concurrent.cu3.web" to the following would cause conflict with app-defined ManagedExecutorService.
         // There is a spec proposal to detect conflict and avoid automatically adding the bean.
         ShrinkHelper.defaultDropinApp(server, APP_NAME_EE10, "concurrent.cdi4.web");
+
+        // Create concurrentCDIWeb2.war
+        ShrinkHelper.defaultDropinApp(server, APP_NAME_EE10_2, "concurrent.cdi.web2");
```

Comment:
along with prior comment,
```suggestion
        // Create concurrentCDI4WebApp.war
        ShrinkHelper.defaultDropinApp(server, WEBAPP_NAME_EE10, "concurrent.cdi4.webapp");
```